internal/store/common_linux_test.go

```diff
@@ -0,0 +1,125 @@
+package store
+
+import (
+	"context"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	blueprintPkg "github.com/fornellas/resonance/internal/blueprint"
+	"github.com/fornellas/resonance/internal/diff"
+	hostPkg "github.com/fornellas/resonance/internal/host"
+	"github.com/fornellas/resonance/log"
+	resourcesPkg "github.com/fornellas/resonance/resources"
+)
+
+func getTestBlueprint(t *testing.T, ctx context.Context) *blueprintPkg.Blueprint {
+	resources := resourcesPkg.Resources{
+		&resourcesPkg.File{
+			Path: "/tmp/foo",
+			Uid:  123,
+			Gid:  456,
+		},
+		&resourcesPkg.APTPackage{
+			Package: "foo",
+			Version: "1.2.3",
```

Comment:
Hard coded version? Maybe we should centralize’em for testing purposes making it easier to share between tests.

Reply:
In this case, this is just a random value, that can actually be... anything, and the test will still pass. We just neeed "some blueprint", to exercise the store save / load cycle.

In other cases, I agree with you, I've been adding variables for such things (eg: [here](https://github.com/fornellas/resonance/pull/116/files#diff-d1e771ca15910240ac37c43a3ec5807ccb55f896e3530f77b65d869d96d7b0cfR23))